Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,38 @@
+
+#### Playbooks
+
+##### New: Suspicious execution from tmp folder
+
+This playbook addresses the following alerts:
+ 
+- Suspicious process execution from tmp folder
+- Suspicious interactive execution of a binary from the tmp folder
+- Suspicious cron job task execution of a binary from the tmp folder
+- A web server process executed an unpopular application from the tmp folder
+ 
+##### Playbook Stages:
+  
+Analysis: 
+ 
+- Check target process hash reputation
+- Check commandline extracted indicators reputation
+ 
+Investigation:
+ 
+- Search for suspicious related alerts
+ 
+Remediation:
+ 
+- Terminate causality process
+- Quarantine the malicious process image file.
+
+<~XSIAM> (Available from Cortex XSIAM 8.8.0).</~XSIAM>
+
+
+#### Triggers Recommendations
+
+##### New: Suspicious execution from tmp folder
+
+- New: This trigger is responsible for handling the 'Suspicious interactive execution of a binary from the tmp folder', 'Suspicious cron job task execution of a binary from the tmp folder' and 'A web server process executed an unpopular application from the tmp folder' alerts via the 'Suspicious execution from tmp folder' playbook.
+<~XSIAM> (Available from Cortex XSIAM 8.8.0).</~XSIAM>
```

Comment:
there is XSIAM version 8.8

Reply:
The Cortex XSIAM version is not correct.